module/Application/src/Application/Module.php

```diff
@@ -23,17 +23,6 @@ public function onBootstrap(MvcEvent $e)
 
     public function getConfig()
     {
-        return include __DIR__ . '/config/module.config.php';
-    }
-
-    public function getAutoloaderConfig()
-    {
-        return array(
-            'Zend\Loader\StandardAutoloader' => array(
-                'namespaces' => array(
-                    __NAMESPACE__ => __DIR__ . '/src/' . __NAMESPACE__,
-                ),
-            ),
-        );
+        return include __DIR__ . '/../../config/module.config.php';
```

Comment:
Can you help me to understand? :) Thanks
https://github.com/doctrine/DoctrineModule/blob/master/src/DoctrineModule/Module.php#L74